### DISCUSSION TITLE
Version To Run Natively On Apple Silicon

### DISCUSSION THREAD

**Discussion by lifeisbard** [atom-ide-community/atom#425] [Ideas]
Now that you've upgraded Electron to v12, would you be willing to build a binary version using the Electron universal binary or the dedicated Apple silicon Electron binary so those of us with Apple Silicon (i.e. M1, M2, etc.) may run Atom-Community natively? It would give us a performance boost. Several of us have been anxious to run Atom natively. It would be greatly appreciated if you could add this to your the of binaries.

Thank you so much for your attention and please continue the good work!

**Comment by mauricioszabo** [comment]
We actually don't have Electron v12 upgrade on `master` branch yet, but I do have some binaries I did with Electron-Builder here: https://github.com/AtomRebirth/atom/tree/build-without-scripts

Some people said that these are not working for some reason. I'm not familiar with Apple M1, nor I have one available to test, so if you want to debug what's wrong, feel free to clone it and ping me with suggestions. This version does not use the `./script/bootstrap` code so it should work with Node.JS 16 (the LTS version, but _not 17 or 18_ because these changed the C API)

**Reply by mhzawadi** [reply]
Just an FYI, I have tried to build on an M1 and I cant as node 12.18 doeant have a darwin-arm64 package
